src/workers/commentMentionMail.ts

```diff
@@ -64,3 +66,17 @@ export const sendEmailToMentionedUser = async (
   });
   logger.info('comment mention email sent to: ' + commenter.id);
 };
+
+interface Data {
+  commentMention: ChangeObject<CommentMention>;
+}
+
+const worker: Worker = {
+  subscription: 'comment-mention-mail',
```

Comment:
Just raising a question: Should we not opt to retroactively fix the subscription names?

Reply:
I'll fix it